autoprotocol/instruction.py

```diff
@@ -1373,6 +1500,8 @@ class Provision(Instruction):
       Destination(s) for specified resource, together with volume information
     measurement_mode : str
       Measurement mode. Possible values are :py:class:`autoprotocol.constants.MEASUREMENT_MODES`
+    informatics : list(Informatics)
+      List of expected aliquot effects at the completion of this instruction
```

Comment:
It is unfortunate if we need to explicitly define this for every Instruction. As currently defined, this should be implicitly available to all Instructions. Its probably reasonable to move the documentation to the parent-class init method. I.e. in `Instruction` itself.

If we need/want the finer level of control over which Instruction should have informatics, would prefer a mix-in class instead. I.e. `InstructionWithInformatics` subclass. But I don't think we need this?

Otherwise... seems like we'll need to do this for all Instructions. If we want to be more future-proof, we could if generalize implicitly passing arguments to the parent class via keyword arguments. I doubt we'll add lots of top-level fields to Instruction though, so we may just need to go through and add `informatics` to every class :/ 
I.e.
```
def __init__(self, resource_id, dests, measurement_mode="volume", **kwargs)
...
super(Provision, self).__init__(
...
**kwargs
)
```

Reply:
Ya, this was the main bottleneck that I had trouble figuring out...